packages/trader/src/Modules/MT5/Components/mt5-personal-details-form.jsx

```diff
@@ -345,6 +359,7 @@ MT5PersonalDetailsForm.propTypes = {
     onSubmit: PropTypes.func,
     residence_list: PropTypes.array,
     value: PropTypes.object,
+    landing_company: PropTypes.object.isRequired,
```

Comment:
what if landing_company is not available?, isnt it going to fallback

Reply:
It is available since `client-store@init` calls it.